addons/hr_recruitment_event/report/hr_applicant_event_report.py

```diff
@@ -0,0 +1,71 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, tools
+
+
+class HrApplicantEventReport(models.BaseModel):
+    _auto = False
+    _name = 'hr.applicant.event.report'
+    _description = 'Applicant Event Report'
+
+    id = fields.Id()
```

Comment:
Why do you need an id ?

Reply:
I don't know it's my first report; i will test without ;)